v2/src/index.css

```diff
@@ -1,8 +1,4 @@
 html, body {
-  width: 400px;
+  min-width: 400px;
```

Comment:
I decided to do that so when Im running it on web it can extend its width, but it still the same when running inside the chrome extension platform
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/47519689/162685355-1e79c0b4-c4a6-4adf-900d-cb7de6282499.png">